client/components/navbar/navbar-notification.service.js

```diff
@@ -21,33 +21,31 @@ class NavbarNotificationService {
   getSubLinks() {
     return this.getMessages()
       .then(messages => messages.map(message => this.convertSubLink(message)))
-      .catch(() => undefined);
+      .catch(() => this.$q.when([]));
```

Comment:
this would not display en error message and you don't have to return a promise inside a promise, it's how chaning works.